### DISCUSSION TITLE
2022-04-27 Weekly Meeting 53 Agenda

### DISCUSSION THREAD

**Discussion by robertlong** [omigroup/omigroup#219] [Weekly Meetings]
Weekly Meeting 53 is happening on Wednesday April 27th, 2022 from 9:00 AM PST(16:00 UTC) - 10:00 AM PST (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda

- Volunteer to take notes? Volunteer to record? 
- Welcome newcomers

## Announcements: 
- [OMI Elections Followup](https://github.com/omigroup/omigroup/discussions/211)
- Schedule our first leadership meeting

## Proposals
- https://github.com/omigroup/omigroup/issues/218#issuecomment-1110224316
- https://github.com/omigroup/media/issues/32
- [OMI attending VKET Summer 2022, and showcasing our OMi-Chan Avatars](https://github.com/omigroup/omi-chan/issues/29)

Working Group Updates (If we have time):

- OMI Website update
- OMI-Chan group
- Scripting group
- glTF Extensions Group
- https://github.com/omigroup/media
- any others?

**Comment by mrmetaverse** [comment]
RE: https://github.com/omigroup/omi-chan/issues/29

Proposing that we attend VKET Summer 2022, and the OMi-Chan Group could create 1-3 avatars to model and rig and showcase at the event. We have 7 days to make the initial payment and then about 4 months to get the avatars done.

**Reply by mrmetaverse** [reply]
created this milestone to track the effort if approved: https://github.com/omigroup/omi-chan/milestone/2

**Comment by mrmetaverse** [comment]
Lets reflect on our VRChat exploration from last Friday after the M3 Avatar Interoperability Demo Day. (it was fun!) 

https://twitter.com/mrmetaverse/status/1517602798527000576?s=20&t=RMjYA0XmjjIfUpI71TKcmQ

![image](https://user-images.githubusercontent.com/63426722/165399722-fd178dcd-5a71-490a-9e20-e95a97771a7e.png)


We should do this again!

**Comment by mrmetaverse** [comment]
LMB @pcsmgnt shared in general: 

For you information, the update on activities for the the Media and Content WG is as follows:

Work has begun on the definition of roadmap (https://github.com/omigroup/media/issues/20) with the next step to undertake the current state analysis (and understand challenges and opportunities) (https://github.com/omigroup/media/issues/22) taking place next week after the OMI Weekly. 

The following have been invited. @Robert @Shadow @MetaMike @evo @mrmetaverse (Jesse, co-chair) @omi-website-subgroup @antpb @humbletim @jin 

Please let me know if there is anyone else I should add, or better still, register your interest direct on the event if you would like to attend. See https://discord.com/events/770382203782692945/968584758848290837


![image](https://user-images.githubusercontent.com/63426722/165562169-00f35385-68a8-4ca4-ad7a-195d55ab19e8.png)

**Comment by mrmetaverse** [comment]
https://github.com/omigroup/omigroup/issues/221 Redeagle proposal for new welcoming system in discord

**Comment by mrmetaverse** [comment]
We are setting up a "leadership meeting" or "operations meeting" for all OMI leads to gather weekly/bi-weekly to talk about more of the operational stuff that generally takes place during our Wednesday meetings. This will free us up to spend more time on Wednesdays doing demos, and exploring what each other are working on. cc @robertlong

**Comment by mrmetaverse** [comment]
https://github.com/omigroup/omigroup/issues/217 - what about avatar build series? :eyes:

**Reply by mrmetaverse** [reply]
https://github.com/omigroup/omigroup/issues/217#issuecomment-1111225698

@madjin volunteered to help!

**Comment by madjin** [comment]
- anytime there's 3 demos to share can host another demo day + world hop
- OMI folks need some custom avatar drip for platforms like vrchat
- instead of status updates every week / check ins can do more engaging and fun stuff
  - lightning talks / demo days / field trips / interop discussions better for general meetings
  - so many cool projects shown off in DMs / twitter, want to give ppl a platform to share
  - more content creation opportunities this way also
  - adminstrative stuff and status updates can be part of leadership meetings
- how do we wanna show up as a community at other places?
  - OMI fashion pins?

**Comment by mrmetaverse** [comment]
RE: https://github.com/omigroup/omigroup/issues/218#issuecomment-1110224316

Robert came across a creator named [Artsy](https://www.artsyvr.com/wander) who has a art style that is performant and would fit our OMI aesthetic. Robert proposes that we engage Artsy to build us a gltf world that is fully open source and able to be imported into any of our worlds (i.e. V-Sekai, Webaverse, ThirdRoom, Hubs, etc.)

**Reply by mrmetaverse** [reply]
this opens up the ability to not only host beautiful events, but also to conduct testing both in and out of OMI

**Reply by mrmetaverse** [reply]
need to confirm that Artsy would be ok with making this fully open source including all models, etc.

**Reply by mrmetaverse** [reply]
@robertlong to reach out

**Comment by mrmetaverse** [comment]
RE: https://github.com/omigroup/media/issues/32

Robert is proposing that we stand up a Open Metaverse Mastadon.

**Comment by mrmetaverse** [comment]
OMI-Chan group did not meet yesterday. We are hesitant to add work to the group due to spotty turnout. We will give the group the next week to consider their capabilities and then either sunset the group, or restructure moving forward. 

Note: This is perfectly normal. We work with intent and iteratively. Life happens, and sometimes people are no longer available for efforts. We do not shame anyone for this - ever. Sunsetting a group is not permanent, it just means we deprioritize the group until there is enough interest and availability in our community to potentially resume the work. That is part of our working group model and nothing to be ashamed of.
